tests/e2e/standard/enketo/delivery.wdio-spec.js

```diff
@@ -0,0 +1,176 @@
+const moment = require('moment');
+const utils = require('../../../utils');
+const gatewayApiUtils = require('../../../gateway-api.utils');
+const loginPage = require('../../../page-objects/login/login.wdio.page');
+const commonPage = require('../../../page-objects/common/common.wdio.page');
+const placeFactory = require('../../../factories/cht/contacts/place');
+const userFactory = require('../../../factories/cht/users/users');
+const contactPage = require('../../../page-objects/standard/contacts/contacts.wdio.page');
+const reportsPage = require('../../../page-objects/reports/reports.wdio.page');
+const analyticsPage = require('../../../page-objects/analytics/analytics.wdio.page');
+const genericForm = require('../../../page-objects/forms/generic-form.wdio.page');
+const delivery = require('../../../page-objects/standard/forms/delivery.wdio.page');
+
+describe('Delivery', () => {
+  const places = placeFactory.generateHierarchy();
+  const healthCenter = places.find(place => place.type === 'health_center');
+  const user = userFactory.build({ place: healthCenter._id, roles: ['district_admin'] });
+  const pregnantWoman1 = 'Woman1';
+  const pregnantWoman2 = 'Woman2';
+  let medicIDW1 = '';
+  let medicIDW2 = '';
+
+  before(async () => {
+    await utils.saveDocs([...places]);
```

Comment:
```suggestion
    await utils.saveDocs(places);
```

Maybe I am missing something here, but is there a reason to create a new array before passing it to saveDocs?